### ISSUE TITLE
UPDATE FOR RECAPTCHA ISSUES AAA problem with the Amazoncaptcha library!!

### ISSUE THREAD

**Issue by drderre** [Hari-Nagarajan/fairgame#598]
While searching on web and on Amazoncaptcha page i found an issue with the code, in fact the prompt  print not solved everytime it found a captcha (i update the titles of captcha for amazon.it) and maybe there is a trouble in how the code use amazoncaptcha from amazoncaptcha, so here a solution from the developer of Amazoncaptcha. The only question is if we can adapt this to the amazon.py in fairgame.
`from webdriver_manager.chrome import ChromeDriverManager
from amazoncaptcha import AmazonCaptcha
from selenium import webdriver

driver = webdriver.Chrome(ChromeDriverManager().install())
driver.get('https://www.amazon.com/errors/validateCaptcha')

captcha_link = driver.page_source.split('<img src="')[1].split('">')[0]   # extract captcha link
captcha = AmazonCaptcha.fromlink(captcha_link)                            # pass it to `fromlink` class method
print(captcha.solve())`

**Comment by DakkJaniels** [comment]
I have no idea what you are saying or what you think this code will fix. Can you link me to where the AmazonCaptcha dev says this is a fix? It looks like a mishmash of code using the requests and webdriver endpoints.

**Comment by DakkJaniels** [comment]
Also stop opening new issues with your captcha problems

**Comment by drderre** [comment]
![Screenshot (8)](https://user-images.githubusercontent.com/81035022/111920407-0859ee80-8a8f-11eb-8ae6-30973cdb625b.png)

**Comment by drderre** [comment]
here the screen, i wanted only a help for my  troubles, i m sorry

**Comment by DakkJaniels** [comment]
You are going to make me type in the url from your screenshot? 

You don't need to open 3+ issues for the same problem. Just update your existing issue.

**Comment by drderre** [comment]
https://github.com/a-maliarov/amazoncaptcha/issues/27

here' s tthe url.. i repeat i m sorry i m new here...

**Comment by DakkJaniels** [comment]
Thank you, that is much more helpful. I'll look into it.

**Comment by drderre** [comment]
let me know if it woul;d be helpful

**Comment by astrorya** [comment]
I had a somewhat related issue where when the bot would go to purchase a product, it would ask to sign in again and somehow the password would be incorrect, then it would input the right password and ask to solve a captcha and would get stuck in a loop of re-inputting the correct password (I think) into the password box over and over without solving the captcha. Amazon said they were trying to "protect" my account.

**Comment by drderre** [comment]
yeah is a trouble with the captchas mine get stuck everytime one appears

**Comment by sidebeub** [comment]
same here
